src/containers/app/index.js

```diff
@@ -15,7 +16,8 @@ const App = props => (
     <AsyncPizza />
     <Notifications />
     <Switch>
-      <Route path={process.env.REACT_APP_BASEURL} exact component={Home} />
+    <Route path={process.env.REACT_APP_BASEURL} exact component={Home} />
+    <Route path={process.env.REACT_APP_BASEURL + 'informations'} exact component={Informations} />
```

Comment:
Can you make Informations async ? As `Pizza` 🍕

Reply:
Done !